testdata/jennies/rawtypes/disjunctions.txtar

```diff
@@ -360,3 +360,82 @@ func (resource *StringOrBool) UnmarshalJSON(raw []byte) error {
 	return errors.Join(errList...)
 }
 
+-- out/jennies/JavaRawTypes --
+== disjunctions/RefreshRate.java
+package disjunctions;
+
+
+// Refresh rate or disabled.
+public class RefreshRate extends StringOrBool {
+    
+}
+== disjunctions/SomeStruct.java
+package disjunctions;
+
+
+public class SomeStruct {
+    private String Type;
+    private Object FieldAny;
```

Comment:
If we don't generate getters/setters, shouldn't we set these fields as public? :thinking: